src/Raven.Server/Web/System/AdminDatabasesHandler.cs

```diff
@@ -814,7 +836,36 @@ public async Task Delete()
                     }
                 }
 
-                await using (var writer = new AsyncBlittableJsonTextWriter(context, ResponseBodyStream()))
+                await ServerStore.WaitForCommitIndexChange(RachisConsensus.CommitIndexModification.GreaterOrEqual, actualDeletionIndex);
+                var timeout = Server.Configuration.Cluster.OperationTimeout.AsTimeSpan;
+                if(nodesToRemove.Count > 0)
+                    timeout /= nodesToRemove.Count;
```

Comment:
if `nodesToRemove.Count` is 0, 
you won't go inside this loop:
https://github.com/ravendb/ravendb/blob/9bccc81de4766fa07d70b2a6460afc6319df2577/src/Raven.Server/Web/System/AdminDatabasesHandler.cs#L843
**But** you'll get a "System.OverflowException" because of the division of the timeout in 0.

if you have 1 or more nodes to remove the db from, you'll want the timeout of the deletion of all of them will be the `OperationTimeout`, so you give every one of them OperationTimeout/numOfNodes

Reply:
Removed, this whole logic has been moved to separate method